### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by hb020202**

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (snack-attack) [inline .idea/codeStyles/codeStyleConfig.xml:1]: For future reference - I'd add the `.idea` folder to your .gitignore - no need to store this random info about your own individual PHPStorm set up!

Don't worry about it for now 😄

File: dbconn.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+/**Connects to database
+ *
+ * No params given
+ *
+ * @return PDO
+ */
+function connect(){
```
Comment (snack-attack) [inline dbconn.php:8]: missing return type hint

File: dbgetdata.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+/**Pulls info from the database
+ *
+ * @param $connection
+ *
+ * @return mixed
+ */
+function getData($connection) {
```
Comment (snack-attack) [inline dbgetdata.php]: missing return type hint

File: dbgetdata.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    $connection->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $connection->prepare("SELECT `pick_order`, `picker`, `title`, `author`, `genre` FROM `books`;");
```
Comment (snack-attack) [inline dbgetdata.php]: I'd recommend putting your query into a variable and passing it into the function as a second parameter so you can reuse on other pages and with other SQL queries!

File: index.php [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    <h1>Book Club Book Tracker</h1>
+</header>
+<nav>
+    <a class="btn" href="add.php">Add</a>
+    <a class="btn" href="#top">Back to top</a>
+</nav>
+<main id="top">
+    <?php
+    foreach ($data as $value) {
+        echo "<div><p>Book# " . $value['pick_order'] . " - " . $value['picker'] . "'s Pick</p><br><p class='title'>" . $value['title'] . "</p><br>" . $value['author'] . "<br><p class='genre'>" . $value['genre'] . "</p></div>";
```
Comment (snack-attack) [inline index.php]: I'd put every value on a separate line so this block is a bit easier to read

File: index.php [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    <title>Book Club Book Tracker</title>
+    <meta name="author" content="Holly Bedford">
+    <meta name="description" content="Book tracker">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Slab&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style1.css" type="text/css">
+</head>
+<body>
+<header>
```
Comment (snack-attack) [inline index.php]: Watch your indentation in this file! Nested elements get indented inside their parent

**Review by snack-attack — CHANGES_REQUESTED** [review]
This is looking really good Holly!

Just a few suggestions for ya 😄

File: .idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (hb020202) [reply .idea/codeStyles/codeStyleConfig.xml:1]: Forgot I could do that!

File: index.php [hunk 7]
```diff
@@ -0,0 +1,39 @@
+<?php require('dbconn.php');
```
Comment (snack-attack) [inline index.php]: I'd put this require on a new line for consistent formatting 😄

**Review by snack-attack — APPROVED** [review]
Very minor formatting comment, but once that's fixed, I'm happy for you to merge this into master! 

Well done Holly!

**Pull request merged**